src/3-Prayer-Request/prayer-request-widgets.tsx

```diff
@@ -38,91 +46,89 @@ export const PrayerRequestTouchable = (props:{prayerRequestProp:PrayerRequestLis
 
     const styles = StyleSheet.create({
```

Comment:
Looks like in the other existing widgets, `styles` is declared outside the component, statically in the file.  Makes since as it's not dependent on state.  This means it could also be in a separate file, that's my recommendation.